packages/sfc-playground/package.json

```diff
@@ -2,13 +2,15 @@
   "name": "@vue/sfc-playground",
   "version": "3.3.4",
   "private": true,
+  "type": "module",
   "scripts": {
     "dev": "vite",
     "build": "vite build",
     "serve": "vite preview"
   },
   "devDependencies": {
     "@vitejs/plugin-vue": "^4.2.3",
+    "execa": "^7.1.1",
```

Comment:
There's not necessary to add this because we've installed `execa` in the root package and this package will never be published.

Reply:
understood, pulling it out.

Reply:
updated as requested see: 5541c3a